src/components/DateRangePicker.jsx

```diff
@@ -125,9 +136,44 @@ export default class DateRangePicker extends React.Component {
     const { onFocusChange } = this.props;
     if (!this.isOpened()) return;
 
+    this.setState({
+      isDateRangePickerInputFocused: false,
+      isDayPickerFocused: false,
+      showKeyboardShortcuts: false,
+    });
+
     onFocusChange(null);
   }
 
+  onDateRangePickerInputFocus(focusedInput) {
+    const { onFocusChange, withPortal, withFullScreenPortal } = this.props;
+
+    if (focusedInput) {
+      const moveFocusToDayPicker = withPortal || withFullScreenPortal;
```

Comment:
Would it make more sense to target based on touch instead of portal presence? The mobile vertical scrollable date picker doesn't use these portal props, but it also requires daypicker focus.